add-chain/cmd/promote-to-standard.go

```diff
@@ -37,8 +37,8 @@ var PromoteToStandardCmd = cli.Command{
 
 		superchainRepoPath := filepath.Dir(filepath.Dir(filepath.Dir(thisFile)))
 		targetDir := filepath.Join(superchainRepoPath, "superchain", "configs", chain.Superchain)
-		targetFilePath := filepath.Join(targetDir, chain.Chain+".yaml")
-		err := config.WriteChainConfig(*chain, targetFilePath)
+		targetFilePath := filepath.Join(targetDir, chain.Chain+".toml")
+		err := config.WriteChainConfigTOML(*chain, targetFilePath)
```

Comment:
Not a blocker for this PR and out of scope, but I would like to fix this to work in a more lightweight way. As is, it pulls hardfork overrides in to the leaf configs.

Reply:
https://github.com/ethereum-optimism/superchain-registry/issues/347